guides/source/action_cable_overview.md

```diff
@@ -767,6 +767,15 @@ config.action_cable.disable_request_forgery_protection = true
 By default, Action Cable allows all requests from localhost:3000 when running
 in the development environment.
 
+In some cases you may need to add more origins to allow requests in development depending on the environment.
+
+```ruby
+# config/environments/development.rb
+config.action_cable.allowed_request_origins = [/http:\/\/*/, /https:\/\/*/, /file:\/\/*/, 'file://', nil]
```

Comment:
Shouldn't these regexes be more strict?
```suggestion
config.action_cable.allowed_request_origins = [/\Ahttp:\/\/*/, /\Ahttps:\/\/*/, /\Afile:\/\/*/, 'file://', nil]
```
I'm also a bit scared this will make people vulnerable to DNS rebinding attacks.
https://blog.saeloun.com/2019/10/31/rails-6-adds-guard-against-dns-rebinding-attacks.html

Reply:
And does `nil` basically mean allow all?

Reply:
@p8 That's a good question, I think `nil` does allow all. My assumption is that this option is used for CORS, but not sure if that limits the blast-zone here.

I think people still might want to "allow all" in some cases that were mentioned in #31524, but perhaps its better to put this in a `WARNING:` as a "user beware" that you can do this but you probably don't want it on by default.

Reply:
I think adding a `WARNING:` is a good idea 👍 

Looking at line 757 to 765, it seems a solution is already in place for the `nil` solution:
```config.action_cable.disable_request_forgery_protection = true```

Line 774 seems like a repetion of line 758.